docs/source/guides/metrics.rst

```diff
@@ -0,0 +1,168 @@
+=======
+Metrics
+=======
+
+Metrics are measurements of statistics about your service, which can provide information about the usage and performance of your bentos in production.
+
+BentoML allows users to define custom metrics with `Prometheus <https://prometheus.io/>`_ to easily enable monitoring for their Bentos.
+ 
+This article will dive into how to add custom metrics to monitor your BentoService and how you can incorporate custom metrics into a :ref:`concepts/runner:Custom Runner`.
```

Comment:
Can we expand the example to include API server as well?

Reply:
I think API server are similar to runner implementation. But I can add it in for the sake of completeness.